packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -244,16 +244,21 @@ class ValidatedLinkDatas {
   final List<LinkData> byPath;
 }
 
+/// Represents a path in a deep link.
 class Path {
   Path({
     required this.path,
     this.queryParams = const {},
     this.isExcluded = false,
   });
+
   final String path;
 
   // TODO(hangyujin): display queryParams in path table.
   final Map<String, String> queryParams;
+
+  // A Boolean value that indicates whether to stop pattern matching and prevent the universal
+  // link from opening if the URL matches the associated pattern. The default is false.
   final bool isExcluded;
```

Comment:
would it be possible to flip this logic so that this field is `include`? That would make it much easier to reason about when using boolean inversions

Reply:
It is `exclude` in AASA [Properties](https://developer.apple.com/documentation/bundleresources/applinks/details/components#properties) 
and `isExcluded` in ads APIs,

I think it makes sense to keep consistent with them, especially the AASA standard